.github/workflows/ci.yml

```diff
@@ -37,3 +37,22 @@ jobs:
       - uses: ./.github/actions/setup
       - name: Build Docs
         run: pnpm run build
+
+  sonarcloud:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: 'Run SonarCloud scanner'
+        if: ${{contains(github.ref, '/pull/')}}
+        uses: SonarSource/sonarcloud-github-action@v2.2.0
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

Comment:
Token needs to be accessible or replaced with an accessible one.

Reply:
@minvws/irealisatie-operations kunnen jullie daar naar kijken